xskillscore/tests/conftest.py

```diff
@@ -140,3 +140,9 @@ def weights_dask(weights):
     Weighting array by cosine of the latitude.
     """
     return weights.chunk()
+
+
+@pytest.fixture
```

Comment:
So you think it should be optional? I would definitely put it optional when a probability because there we know the bounds 0 and 1. Here the edges could be any numbers as in contingency

Reply:
Sorry, I got myself confused and meant to remove this comment. I agree that `caregory_bins`should be required